meeteval/io/keyed_text.py

```diff
@@ -50,3 +50,6 @@ def merged_transcripts(self) -> str:
     def utterance_transcripts(self) -> List[str]:
         """There is no notion of an "utterance" in CTM files."""
         raise NotImplementedError()
+
+    def filenames(self):
```

Comment:
Add this function to the base class?

Reply:
Can we be sure that all `BaseLine` classes have a filename?

Reply:
I think all classes have the concept of filename, otherwise you cannot find the mapping between ref and hyp

Reply:
Done